packages/studio-base/src/components/MessagePipeline/index.tsx

```diff
@@ -335,3 +335,8 @@ function createPlayerListener(args: {
     },
   };
 }
+
+export const getTopicToSchemaNameMap = (
```

Comment:
Couldn't this be in another file?  I don't think this is a core function of "MessagePipeline" to be in the index.tsx. Also this is used on the PanelSettings component, would it not make more sense to be there? Also, if it grows a little, it has its own file to do extra stuff.

Also, could you create a unit test for this as well? It may grow in the future or even be reimplemented, and it would be good to keep its behaviour